src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -1698,74 +1586,11 @@ ALTER TABLE test_table ADD COLUMN id2 int DEFAULT nextval('mx_test_sequence_1');
 ALTER TABLE test_table ALTER COLUMN id2 DROP DEFAULT;
 ALTER TABLE test_table ALTER COLUMN id2 SET DEFAULT nextval('mx_test_sequence_1');
 SELECT unnest(master_metadata_snapshot()) order by 1;
```

Comment:
How hard is it to keep this UDF with all the commands that `activate_node` generates? If trivial, I'd be in favor of keeping it for debugging purposes.